### DISCUSSION TITLE
Calibration Button missing

### DISCUSSION THREAD

**Discussion by Maraaaaaaaaaaa** [NightscoutFoundation/xDrip#1960] [Q&A]
Hi, I've been having problems with this for months now. Everytime I stop a sensor (Libre 2) and start a New one, the calibration button just disappears.
Everytime this issue happens it's a saga to get It back to work. I try reseting the sensor multiple times, uninstall the app, clear cache, clear ALL data, restart the phone. Sometimes I have to uninstall and install the app 6 or 7 times to get It to work. I've tried to find a pattern that could explain why It disappears and why after all that process of cleaning data, cache, uninstalling, restarting the phone, at some point the button appears again, but I wasn't able to figure it out. 
Please help, it's really hard to do all of that everytime.

PS.: I've tried downloading diferent versions of the apk, pre-releases and releases. The versions doesn't seem to matter.

![Screenshot_20220128-085924~2](https://user-images.githubusercontent.com/98590933/151543707-8e2a8a7a-f112-4242-9b7d-59b95fe1ef78.png)

More info: 
Android: 11
Phone: motorola one vision

**Comment by Navid200** [comment]
I don't think the calibration button is supposed to be there before you have had 3 readings after you start a sensor.

**Comment by tzachi-dar** [comment]
@Maraaaaaaaaaaa how long have you been waiting for it to appear?
Do you see that the sensor gets connected on the status screen?
Do you see a message saying "data was recieved?"

I think that after waiting something like 20 minutes it should appear.

Also, make sure that you scan it and wait for about 1 more second after you see scan succeeded.

There is a PR https://github.com/NightscoutFoundation/xDrip/pull/1961 that should give us more information about what is going on there.

If after 30 minutes it still does not appear, please send your DB to tzachi.dar@gmail.com and I'll try to understand what the issue is.

**Reply by Maraaaaaaaaaaa** [reply]
Ok, I'll try to do those things and get back to you guys.
